### DISCUSSION TITLE
fingerbot Adaprox TS0001 not interviewed

### DISCUSSION THREAD

**Discussion by SimonBard** [Koenkk/zigbee2mqtt#24037] [Devices]
I have recently bought two fingerbots, one from[ aliexpress ](https://de.aliexpress.com/item/1005005715461824.html?) and one from [amazon](https://www.amazon.de/dp/B0CXTBH2KZ).
Both are not interviewed by zigbee2mqtt. They are in pairing mode, so they are both blinking. Zigbee2mqtt is up to date. I use CC2531, there is no new firmware for this, but I guess it should still work properly.

**Comment by SimonBard** [comment]
push* 

Can you give me a link where to buy a fingerbot? Mine is not even interviewed.

**Reply by lux73** [reply]
these https://de.aliexpress.com/item/1005006294685094.html working smooth and flawless for me

z2m 1.40.1 / CC2652RB FW20240710

btw CC2531 is in 2024 considered as crap -> i suggest to replace with CC2652x Controller

**Reply by SimonBard** [reply]
Many thanks. Can you give me also a link to your  CC2652RB  controller? Looked on aliexpress and ebay, but nothing to find with a usb

**Reply by lux73** [reply]
sure - it's linked on the official z2m Website -> https://www.zigbee2mqtt.io/guide/adapters/zstack.html#recommended-cc2652-cc1352

[Slaesh's CC2652RB stick](https://slae.sh/projects/cc2652/)

i really can recommend this Stick - but generally any other CC2652x based Coordinator should do
